draft-ietf-quic-transport.md

```diff
@@ -2391,13 +2387,16 @@ the packet other than the last 16 bytes for carrying data.
 
 ### Detecting a Stateless Reset
 
-An endpoint detects a potential stateless reset when a incoming packet
-with a short header either cannot be associated with a connection,
-cannot be decrypted, or is marked as a duplicate packet.  The endpoint
-then compares the last 16 bytes of the packet with the Stateless Reset
-Token provided by its peer, either in a NEW_CONNECTION_ID frame or
-the server's transport parameters.  If these values are identical,
-the endpoint MUST enter the draining period and not send any further
+An endpoint detects a potential stateless reset when a incoming packet with a
+short header either cannot be associated with a connection, cannot be decrypted,
+or is marked as a duplicate packet.  The endpoint MUST then compare the last 16
+bytes of the packet with all Stateless Reset Tokens provided by its peer, either
```

Comment:
Is there a potential for abuse here: all stateless resets could be a lot?

Reply:
That's a risk you take when you send packets with lots of different connection IDs.  I notice that I don't mention those that are retired though.  I'll fix that.